lib/stripe/recipient_transfer.rb

```diff
@@ -0,0 +1,4 @@
+module Stripe
+  class RecipientTransfer < APIResource
```

Comment:
Actually, looking at this one more time, you don't need anything on `APIResource` and some of its methods like `#refresh` aren't going to work for `RecipientTransfer`. Can you inherit from `StripeObject` instead? Thanks!

ptal @remi-stripe